octokit/search_test.go

```diff
@@ -0,0 +1,123 @@
+package octokit
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSearchService_UserSearch(t *testing.T) {
+	setup()
+	defer tearDown()
+
+	mux.HandleFunc("/search/users", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		respondWithJSON(w, loadFixture("user_search.json"))
+	})
+
+	url, err := SearchURL.Expand(map[string]interface{}{
+		"type":  "users",
+		"query": "dhruvsinghal"})
+	assert.NoError(t, err)
+
+	searchResults, result := client.Search(url).UserSearch()
```

Comment:
What do you think of naming the method to `client.Search(url).Users()`? It seems to read more fluently.